client/signup/steps/design-picker/index.jsx

```diff
@@ -156,15 +165,24 @@ export default function DesignPickerStep( props ) {
 						align="left"
 					/>
 				}
-				categoriesFooter={
-					useFeaturedPicksButtons && (
-						<FeaturedPicksButtons designs={ featuredPicksDesigns } onSelect={ pickDesign } />
-					)
-				}
+				categoriesFooter={ renderCategoriesFooter() }
 			/>
 		);
 	}
 
+	function renderCategoriesFooter() {
+		return (
+			<div>
```

Comment:
Nit pick: could we use a React fragment here instead? i.e. `<> ... </>`. That way we're not introducing an unnecessary layer of DOM nesting.